pkg/Tools-Override/_

```diff
@@ -0,0 +1 @@
+
```

Comment:
we should just remove the tools-override directory and part of the script.

Reply:
Even though that makes it different from the corefx version?

Reply:
It just feels pointless to add this empty file. You could also add a exists check in the scripts perhaps.

Reply:
I don't have a strong opinion but I could definitely see someone looking at this in the future and asking what it is for and even try deleting it.

Reply:
Ok, the exists check sounds good. I'll do that.